### DISCUSSION TITLE
How to select almost all fields and except some fields.

### DISCUSSION THREAD

**Discussion by l2D** [prisma/prisma#7773] [Q&A]
Hello Prisma,

I want to know How to select almost all fields and except some fields on the table.

Example:

```javascript
const result = await prisma.user.findUnique({
  where: { id: 1 },
  select: {
    except: [field_name_1, .... , field_name_N]
  },
})
```

[`select` model query](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#select)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
